.vscode/launch.json

```diff
@@ -13,6 +13,15 @@
       "args": ["--extensionDevelopmentPath=${workspaceFolder}/packages/vsce"],
       "outFiles": ["${workspaceFolder}/packages/vsce/out/**/*.js"],
       "preLaunchTask": "npm: watch"
+    },
+    {
+      "name": "Debug Zowe SSH Command",
+      "type": "node",
+      "request": "launch",
+      "program": "${workspaceFolder}/node_modules/@zowe/cli/lib/main.js",
```

Comment:
very clever way to import the CLI as a devDep and use it for debugging purposes 🥳